content/control-api/index.textile

```diff
@@ -0,0 +1,229 @@
+
+<a href="/images/screenshots/control-api/account-settings-menu-item.png" target="_blank">
+  <img src="/images/screenshots/control-api/account-settings-menu-item.png" style="width: 40%" alt="Account Settings">
+</a>
+
+Your Account Settings are displayed.
+
+h3(#app-id). How to find your Application ID
+
+In the "Ably dashboard":https://ably.com/accounts/any select the app you want to find the application ID for. Click on the "Settings" tab:
+
+<a href="/images/screenshots/control-api/application-settings.png" target="_blank">
+  <img src="/images/screenshots/control-api/application-settings.png" style="width: 70%" alt="Application Settings">
+</a>
+
+The App ID is displayed under "Application settings". It is also the first part of your API key for that application. For example, if your API key is @28AB6c.DEFi0Q@, then the App ID is @28AB6c@.
+
+h2(#quickstart). Quickstart
+
+In the code examples, you will need to set the following variables by any convenient method (such as setting the variables in a script, or copying and pasting suitable values directly into the code):
+
+|_. Variable |_. Description |
+| ACCOUNT_ID | Your Ably Account ID (see "here":#account-id) |
+| ACCESS_TOKEN | Your Ably access token for the Control API (see "here":#authentication) |
+| APP_ID | The application ID for the application you want to modify (see "here":#app-id) |
+
+h3(#quickstart-create-application). Create an application
+
+To create an application:
+
+```[sh]
+curl --location --request POST 'https://control.ably.net/v1/accounts/${ACCOUNT_ID}/apps' \
+--header 'Content-Type: application/json' \
+--header 'Authorization: Bearer ${ACCESS_TOKEN}' \
+--data-raw '{
+    "name": "Created App 123",
+    "status": "enabled",
+    "tlsOnly": true,
+    "fcmKey": null,
+    "apnsCertificate": null,
+    "apnsPrivateKey": null,
+    "apnsUseSandboxEndpoint": false
+}'
+```
+
+Sample response:
+
+```[json]
+{
+    "accountId": "VgQpOZ",
+    "id": "bh4QSw",
+    "name": "Created App 123",
+    "status": "enabled",
+    "tlsOnly": true,
+    "apnsUseSandboxEndpoint": false,
+    "created": 1625813276973,
+    "modified": 1625813276973
+}
+```
+
+h3(#quickstart-list-application). List applications
+
+To list all the apps associated with your account:
+
+```[sh]
+curl "https://control.ably.net/v1/accounts/${ACCOUNT_ID}/apps" \
+     --header "Authorization: Bearer ${ACCESS_TOKEN}" \
+     --header "Accept: application/json"
+```
+
+Sample response:
+
+```[json]
+[
+  {
+      "accountId": "VgQpOZ",
+      "id": "bh4QSw",
+      "name": "Created App 123",
+      "status": "enabled",
+      "tlsOnly": true,
+      "apnsUseSandboxEndpoint": false,
+      "created": 1625813276973,
+      "modified": 1625813276973
+  },
+  ...
+]
+```
+
+h2(#rate-limits). Rate limits
+
+The Control API has certain limits on the number of API calls that can be made per hour. The limits are as follows:
```

Comment:
it would be nice to link to our new limits article here?

Reply:
Yes, but we'll have to wait on that one.